op-chain-ops/crossdomain/migrate.go

```diff
@@ -77,12 +79,15 @@ func MigrateWithdrawal(withdrawal *LegacyWithdrawal, l1CrossDomainMessenger *com
 		return nil, fmt.Errorf("cannot abi encode relayMessage: %w", err)
 	}
 
+	// Set the outer gas limit. This cannot be zero
+	gasLimit := uint64(len(data)*16 + 200_000)
```

Comment:
Would be good to document this + determine if it's a sufficient gas limit. LGTM otherwise.

Reply:
We have a notion doc explaining the rationale, the plan is to merge and then test e2e